### PR TITLE
Removed all unmangled functions.

### LINKED ISSUE #1502: Remove unmangled from kean repo
External functions defined by unmangled should not be defined in this repo.

### PR DESCRIPTION
**Pull request by erikhagglund**
It is now up to the user of ooc-kean to specify which functions it wants to unmangle and expose externally

Fixes #1502

@emilwestergren

### CODE REVIEW

File: source/system/Exception.ooc [hunk 1]
```diff
@@ -200,9 +200,6 @@ OutOfBoundsException: class extends Exception {
 	init: func ~noOrigin (accessOffset, elementLength: Int) {
 		message = "Trying to access an element at offset %i, but size is %i!" format(accessOffset, elementLength)
 	}
-	kean_exception_outOfBoundsException_throw: static unmangled func (accessOffset, elementLength: Int) {
```
Comment (sebastianbaginski) [inline source/system/Exception.ooc]: This function is called from Array.h in safe version https://github.com/magic-lang/ooc-kean/blob/master/source/system/Array.h#L13 , make sure it works when building with -Dsafe.

Comment (marcusnaslund) [inline source/system/Exception.ooc]: Yeah, this one should probably stay, it's not related to other projects.

Comment (emilwestergren) [inline source/system/Exception.ooc]: Yes this should stay

Comment (erikhagglund) [inline source/system/Exception.ooc]: yes :) it was mistakenly removed, re-added it again.

@marcusnaslund @sebastianbaginski @emilwestergren

**Comment by marcusnaslund** [comment]
This should be merged together with https://github.com/magic-lang/ooc-kean/pull/1507, right?

**Comment by marcusnaslund** [comment]
Put on hold until corresponding PR is ready.

**Comment by marcusnaslund** [comment]
Conflicts.

**Comment by marcusnaslund** [comment]
This has the added benefit of bumping up the test coverage percentage, nice. :)

**Pull request merged**